pkg/cover/backend/elf.go

```diff
@@ -315,9 +410,10 @@ func symbolize(target *targets.Target, objDir, srcDir, buildDir, obj string, pcs
 		for _, frame := range res.frames {
 			name, path := cleanPath(frame.File, objDir, srcDir, buildDir)
 			frames = append(frames, Frame{
-				PC:   frame.PC,
-				Name: name,
-				Path: path,
+				Module: frame.Module,
```

Comment:
Rather than extending symbolizer.Frame with Module, I think  it will be cleaner to convert symbolizer.Frame to backend.Frame right in the loop above, or extend symbolizerResult with Module name.
Extending symbolizer.Frame with module looks like a work-around some quirk somewhere else. We already give symbolize the binary name, why do we give it some random string (which does not even need to be module name) just to return it back to us?